generator/test_case_generator.go

```diff
@@ -44,13 +45,14 @@ type testConfig struct {
 	runMockServer bool
 	// define target matrix field as set(s)
 	// empty map means a testConfig will be created with a test entry for each entry from *_test_matrix.json
-	targets map[string]map[string]struct{}
+	targets map[string]map[any]struct{}
```

Comment:
hop limit is an int and not a string

Reply:
I was curious if it were plausible and better practice to parse hop_limit into a string, but it makes sense to validate hop_limit as an int.